app/services/applications/change_funding_eligibility.rb

```diff
@@ -29,7 +29,11 @@ def change_funding_eligibility
       funding_eligiblity_status_code =
         (eligible_for_funding ? :marked_funded_by_policy : :marked_ineligible_by_policy)
 
-      application.update!(eligible_for_funding:, funding_eligiblity_status_code:)
+      application.assign_attributes(eligible_for_funding: eligible_for_funding, funding_eligiblity_status_code: funding_eligiblity_status_code)
```

Comment:
Presumably you can still use the hash shorthand here (as was used when this was (`#update!`) ?

Reply:
yes - not sure why rubocop didn't pick this up.
I've switched back to the original `update!` method, as I can use `saved_changes` (thanks for the suggestion).